features/general.feature

```diff
@@ -71,7 +71,7 @@ Feature: General tests of WP Launch Check
 
   Scenario: WordPress has a new major version but no new minor version
     Given a WP install
-    And I run `wp core download --version=5.6.11 --force`
+    And I run `wp core download --version=4.6.27 --force`
```

Comment:
This drop from 5.6.x to 4.6.x is continuing to cause Behat tests to fail. We need to figure out where this is coming from.